core-services/prow/02_config/_config.yaml

```diff
@@ -21,6 +21,15 @@ plank:
         mediaTypes:
           log: text/plain
       gcs_credentials_secret: gce-sa-credentials-gcs-publisher
+    "openshift/private-ci-testing":
+      gcs_configuration:
+        bucket: origin-ci-private
+        path_strategy: single
```

Comment:
we can remove this and the defaults if we want, for a cleaner dir structure with no implicit dirs